Citadel/Putricide.lua

```diff
@@ -128,6 +130,11 @@ do
 		self:Bar("phase", 18, L["phase_bar"], "achievement_boss_profputricide")
 		self:ScheduleTimer(nextPhase, 3)
 		stopOldStuff()
+		if not first then
+			self:SetStage(2)
```

Comment:
Why not just place these with the messages of stage 2 and stage 3? One would expect these to be when we announce them in the messages.